### PR TITLE
hplip-plugin: fix lilac.yaml

### LINKED ISSUE #1634: hplip lilac nvchecker error
### 问题类型 / Type of issues

* 打包错误 / packaging error

### 受影响的软件包 / Affected packages

* （请在此填写包名，每行一个，开头加上星号 / package names, one per line）
hplip
----

请在此补充说明。
Please describe in detail here.

I was asked to review https://github.com/archlinuxcn/repo/pull/1631 since I am the maintainer of hplip. I don't know anything about how lilac works though, so I just merged the pull request thinking that whoever wrote it knew what they were doing. Since then, lilac has been emailing me about the following error:

{"logger_name": "nvchecker.get_version", "name": "hplip-plugin:1", "event": "no idea to get version info.", "level": "error"}
{"logger_name": "nvchecker.core", "name": "hplip-plugin:1", "event": "no-result", "level": "warning"}

Will this error prevent lilac from maintaining the package? If so, I would like someone who knows how to fix this to take over the package.

### PR DESCRIPTION
**Pull request by edward-p**
fix #1634

### CODE REVIEW

**Comment by petronny** [comment]
参与的把自己都加到maintainers里

update_on里加上AUR上游

**Comment by edward-p** [comment]
> 参与的把自己都加到maintainers里
> 
> update_on里加上AUR上游

AUR 上游也是 colinkeenan，不如让 colinkeenan 把 lilac 加入 AUR 的 co-maintainers，让 lilac 自动更新AUR？

File: archlinuxcn/hplip-plugin/lilac.py [hunk 1]
```diff
@@ -7,5 +7,6 @@ def pre_build():
 
 def post_build():
   git_pkgbuild_commit()
+  update_aur_repo()
```
Comment (edward-p) [inline archlinuxcn/hplip-plugin/lilac.py:10]: Add lilac as co-maintainer on AUR to let lilac update the AUR repo for you.

Comment (hubutui) [reply archlinuxcn/hplip-plugin/lilac.py:10]: 我看没啥问题。不过我刚刚 push 了 lilac.yaml 的修改诶。。。才看到这里。

Comment (edward-p) [reply archlinuxcn/hplip-plugin/lilac.py:10]: > 我看没啥问题。不过我刚刚 push 了 lilac.yaml 的修改诶。。。才看到这里。

没事，github好像没检测到冲突，到时候合并应该可以自动解决的？

**Comment by oldherl** [comment]
Why are you all speaking Chinese under a PR about a non-Chinese author?

**Comment by colinkeenan** [comment]
I don't know much about this, but merged assuming you do.

**Comment by yan12125** [comment]
> Why are you all speaking Chinese under a PR about a non-Chinese author?

@hubutui starts using Chinese in https://github.com/archlinuxcn/repo/pull/1631. That is really BAD.

**Pull request merged**